ods_ci/tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -292,6 +292,7 @@ Verify All The Pods Are Using Image Digest Instead Of Tags
     ...       Tier1
     ...       ODS-2406
     ...       ExcludeOnODH
+    ...       ExcludeOnNightly
```

Comment:
Do we plan to use this tag? AFAIK we don't use this in Jenkins anywhere? :thinking:

Reply:
Hi @jstourac , you are right, we do not use it atm. But I plan to set up a disconnected nightly job and having such a tag would allow us to reduce the number of failed tests.